libs/cart/src/effects/cart.effects.spec.ts

```diff
@@ -35,60 +45,111 @@ describe('Daffodil | Cart | CartEffects', () => {
+
+    daffCartStorageSpy.getCartId.and.returnValue(String(mockCart.id));
   });
 
   it('should be created', () => {
     expect(effects).toBeTruthy();
   });
 
   describe('when CartLoadAction is triggered', () => {
-
     let expected;
     const cartLoadAction = new DaffCartLoad();
 
     describe('and the call to CartService is successful', () => {
-
       beforeEach(() => {
-        spyOn(daffDriver, 'get').and.returnValue(of(mockCart));
+        daffDriverSpy.get.and.returnValue(of(mockCart));
         const cartLoadSuccessAction = new DaffCartLoadSuccess(mockCart);
         actions$ = hot('--a', { a: cartLoadAction });
         expected = cold('--b', { b: cartLoadSuccessAction });
       });
 
       it('should dispatch a CartLoadSuccess action', () => {
-        expect(effects.load$).toBeObservable(expected);
+        expect(effects.get$).toBeObservable(expected);
       });
     });
 
     describe('and the call to CartService fails', () => {
-
       beforeEach(() => {
         const error = 'Failed to load cart';
         const response = cold('#', {}, error);
-        spyOn(daffDriver, 'get').and.returnValue(response);
+        daffDriverSpy.get.and.returnValue(response);
         const cartLoadFailureAction = new DaffCartLoadFailure(error);
         actions$ = hot('--a', { a: cartLoadAction });
         expected = cold('--b', { b: cartLoadFailureAction });
       });
 
       it('should dispatch a CartLoadFailure action', () => {
-        expect(effects.load$).toBeObservable(expected);
+        expect(effects.get$).toBeObservable(expected);
       });
     });
   });
 
-  describe('when AddToCartAction is triggered', () => {
+  describe('when CartCreateAction is triggered', () => {
+    let expected;
+    const cartCreateAction = new DaffCartCreate();
+
+    describe('and the call to CartService is successful', () => {
+      beforeEach(() => {
+        daffDriverSpy.create.and.returnValue(of({id: mockCart.id}));
+        const cartCreateSuccessAction = new DaffCartCreateSuccess({id: mockCart.id});
+        actions$ = hot('--a', { a: cartCreateAction });
+        expected = cold('--b', { b: cartCreateSuccessAction });
+      });
+
+      it('should dispatch a CartCreateSuccess action', () => {
+        expect(effects.create$).toBeObservable(expected);
+      });
+    });
+
+    describe('and the call to CartService fails', () => {
+      beforeEach(() => {
+        const error = 'Failed to create cart';
+        const response = cold('#', {}, error);
+        daffDriverSpy.create.and.returnValue(response);
+        const cartCreateFailureAction = new DaffCartCreateFailure(error);
+        actions$ = hot('--a', { a: cartCreateAction });
+        expected = cold('--b', { b: cartCreateFailureAction });
+      });
+
+      it('should dispatch a CartCreateFailure action', () => {
+        expect(effects.create$).toBeObservable(expected);
+      });
+    });
+  });
+
+  describe('when CartCreateSuccessAction is triggered', () => {
```

Comment:
I think this test could be better. Take a look at what I send you in slack. You should be able to test that the store service is actually called.